src/test/scala/org/scalastyle/scalariform/NonASCIICharacterCheckerTest.scala

```diff
@@ -39,8 +39,18 @@ class NonASCIICharacterCheckerTest extends AssertionsForJUnit with CheckerTest {
     assertErrors(List(), source)
   }
 
+  @Test def testStringOK(): Unit = {
+    val source = """
+                   |package foobar
+                   |// non-ascii in string via unicode escape - ok
+                   |class OK {
+                   |  val s = "%s"
+                   |}""".stripMargin.format("\\ud83c\\udf4e")
```

Comment:
If somebody knows how to correctly produce the desired string in a cleaner way I'd be happy to update.

I found that using `\ud83c\udf4e` in triple-quote string resulted in `🍎` (i.e. escape was applied), and using `\\ud83c\\udf4e` resulted in `\\ud83c\\udf4e` (i.e. double back slashes are left in literal form). Unsure how to cleanly get desired outcome of literal `\ud83c\udf4e` within a triple-quote string.